.github/workflows/flatpak-build.yaml

```diff
@@ -0,0 +1,193 @@
+name: Flatpak Build
+on:
+  workflow_call:
+    inputs:
+      arch:
+        default: x86_64
+        type: string
+      build-bundle:
+        default: true
+        type: boolean
+      bundle:
+        default: obs-studio-flatpak.flatpak
+        type: string
+jobs:
+  build-until-90:
+    name: Build Modules Until 90-* 🧾
```

Comment:
A more correct approach would be to be able to explicitly specify which modules to exclude vs using something like `stop-at-module` which depends on the order at which modules are specified. The intent here is to "build everything _but_ this one module", so why this roundabout way of achieving it?

Also doesn't this exclude nlohmann-json and websocketpp as well, as numerically they come after `asio` (another reason why making this based on order is brittle).

At the very least the job should be called something like "Build without CEF" (regardless of its module's filename number) and the filenames "-without-cef" or "-no-cef". IMO it makes more sense to use a name describing the purpose vs describing the implementation detail of achieving the purpose.

Reply:
90-* are headers-only libraries, those are excluded from the first job.

The intent is build until and stop at the first 90-* module which is `asio`

Reply:
As I mentioned that all seems very brittle and entirely based on conventions that are easily broken - we always need to remember not to add any further dependencies that might use any prefix above 90 or we'll break it, especially as the entire intent is to just build without CEF.

I'd rather have either:

* Two separate manifests (one with CEF and one without), though I have a hunch that Flatpak is not designed to allow that, OR
* Enhance `flatpak-builder` to be able to accept one or more "excluded modules" which are then simply skipped (and the values are based on the `name` in the module) than relying on build order and `stop-at-module`

The manifest is already a JSON file (and so are the modules), so why not use their metadata to clearly steer the build process and also express inter-module dependencies that way?

Reply:
- [ ] TODO: More bash code with `jq` to get the first 90-* module

1. Refused by Georges, he asked to improve the action to add support for `stop-at-module` since it already exist inside `flatpak-builder` and use it.
2. `flatpak-builder` has a module order and you can ask it to stop at a module, I think they try to keep the tool simple which is a good thing.

Reply:
Got it - yeah I'm fine with the implementation if this is the only way to achieve it - I would like to think that being able to selectively exclude some modules is a pretty common ask and should be supported directly (I currently don't know what `stop-at-module` could be used for instead).

The main issue with it being based on file name ordering persists (a pattern I'd rather move away from), but that shouldn't hold up this PR.

Reply:
A Flatpak manifest is a recipe, `flatpak-builder` follows the recipe order with a certain strictness but the builder can be asked to stop at a module.
The `--stop-at` is mainly meant  for testing purpose, we just happen to find another usage.
This is not a "packaging system" with dependencies to solve or a tooling like buildstream.

Reply:
Yeah I was afraid of that - we're using features for a purpose they were never meant for and we should rely on such crutches less not more, but alas. I won't block the PR over that.

Reply:
Please open another review thread is the job's name still need to be reviewed. This one became more about the `stop-at-module` determination and the tooling.